### PR TITLE
CentOS 7.0: Fix unattended install

### PR DESCRIPTION
**Pull request by igord-daynix**
1. Fix kickstart file for unattended install.
2. Replace DVD install media by minimal CD.

Signed-off-by: Igor Derzhavets <igor.derzhavets@ravellosystems.com>

### CODE REVIEW

**Comment by igord-daynix** [comment]
Hello,
I will change commit message and re-apply the patch.
Thank you!

**Comment by igord-daynix** [comment]
Please review and commit!

**Comment by waynesun09** [comment]
@igord-daynix pls rebase to resolve conflict, thx

**Comment by igord-daynix** [comment]
@waynesun09 , Resolved, thx!

File: shared/downloads/centos70-64.ini [hunk 1]
```diff
@@ -1,4 +1,4 @@
 [centos70-64]
 title = CentOS 7.0 amd64
-url = http://mirror.isoc.org.il/pub/centos/7.0.1406/isos/x86_64/CentOS-7.0-1406-x86_64-DVD.iso
-destination = isos/linux/CentOS-7.0-1406-x86_64-DVD.iso
+url = http://mirror.isoc.org.il/pub/centos/7.0.1406/isos/x86_64/CentOS-7.0-1406-x86_64-Minimal.iso
```
Comment (waynesun09) [inline shared/downloads/centos70-64.ini]: about this url, should be updated, pls check:
http://mirror.isoc.org.il/pub/centos/7.0.1406/readme

**Comment by igord-daynix** [comment]
@waynesun09,  I have updated URL, ISO name, MD5.

Thanks!

**Pull request merged**